go/libraries/doltcore/sqle/conflicts_tables.go

```diff
@@ -0,0 +1,164 @@
+package sqle
+
+// Copyright 2019 Liquidata, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import (
+	"github.com/liquidata-inc/go-mysql-server/sql"
+
+	"github.com/liquidata-inc/dolt/go/libraries/doltcore/merge"
+	"github.com/liquidata-inc/dolt/go/store/types"
+
+	"github.com/liquidata-inc/dolt/go/libraries/doltcore/doltdb"
+)
+
+const ()
+
+var _ sql.Table = ConflictsTable{}
+
+type ConflictsTable struct {
```

Comment:
You should write some unit tests verifying that you can't insert or update rows in this table. There are a bunch of these for other system tables already. E.g. TestExecuteUpdateSystemTables in sqlupdate_test.go